### PR TITLE
Implemented Aggregate,IntAggregator,Filter,Predicate and JoinPredicate

### PR DESCRIPTION
**Pull request by Ram-Aditya**
#19 #18 
Implemented files of Exercise 1 and Exercise 2 of SimpleDB final project.

### CODE REVIEW

**Review by Priiyam — APPROVED** [review]

**Review by Rithesh17 — APPROVED** [review]

**Comment by sbshah97** [comment]
Why has this Pull Request not been merged?

**Review by manishsbhoopalam — APPROVED** [review]

**Comment by Ram-Aditya** [comment]
@salman-bhai We thought we needed to wait for your approval as we had requested a review from your side.
Sorry about that!We `ll go ahead with the merge now then.

**Pull request merged**
